src/components/StudyRoom/BottomDrawer.js

```diff
@@ -27,27 +27,44 @@ const Puller = styled(Box)(({theme}) => ({
 function SwipeableEdgeDrawer(props) {
     const {window} = props;
     const [open, setOpen] = React.useState(false);
+    const [globalCSS, setGlobalCSS] = React.useState(false);
 
-    const toggleDrawer = (newOpen) => () => {
+    const toggleDrawer = (newOpen) => async () => {
         setOpen(newOpen);
+
+        if (globalCSS) // if global CSS is set (i.e. drawer is open), wait for 300ms until bottom drawer dismisses before removing global CSS
+            await delay(300);
+        if (globalCSS) // disabling global CSS when bottom drawer is closed and enabling it when bottom drawer is open
+            setGlobalCSS(false);
+        else
+            setGlobalCSS(true);
     };
 
+    // method to create delays in the code
+    const delay = ms => new Promise(
+        resolve => setTimeout(resolve, ms)
+    );
+
     // This is used only for the example
     const container = window !== undefined ? () => window().document.body : undefined;
 
     return (
         <Root>
             <CssBaseline/>
-            <Global
-                styles={{
-                    '.MuiDrawer-root > .MuiPaper-root': {
-                        height: `calc(90% - ${drawerBleeding}px)`,
-                        overflow: 'visible',
-                        borderTopLeftRadius: '20px',
-                        borderTopRightRadius: '20px'
-                    },
-                }}
-            />
+            {globalCSS ?
+                <Global
+                    styles={{
+                        '.MuiDrawer-root > .MuiPaper-root': {
+                            height: `calc(90% - ${drawerBleeding}px)`,
+                            overflow: 'visible',
+                            borderTopLeftRadius: '20px',
+                            borderTopRightRadius: '20px'
+                        },
+                    }}
+                />
+                :
+                <></>
+            }
             {/*<Box sx={{ textAlign: 'center', pt: 1 }}>*/}
             <Button style={{width: '100%', height: '100%', background: 'none', border: 'none'}}
                     onClick={toggleDrawer(true)}>{props.icon}</Button>
```

Comment:
why call this function async if you neither await nor use the promise ?

Reply:
Because the delay method is async and I cannot call an asynchronous function inside a synchronous one.